### ISSUE TITLE
Change Browserify by Webpack

### ISSUE THREAD

**Issue by iranreyes** [Jam3/generator-jam3#331]

**Comment by neo** [comment]
I don't think this is gonna happen any time soon, and it won't be easy. But I totally support and vote for it. (according to my limited experience using webpack 😂)

**Comment by jeremenichelli** [comment]
@neo _not happening any time soon..._ hold my beer 🍺

**Comment by neo** [comment]
@jeremenichelli I actually quoted Aaron for that. I personally is 100% *for* webpack
